### PR TITLE
add beta channel advice to the updatenotification section

### PR DESCRIPTION
**Pull request by szaimen**
| Before | After |
|---|---|
| image | image |

Signed-off-by: szaimen <szaimen@e.mail.de>

<details>
<summary>For my own testing</summary>

```
docker run -it \
-e SERVER_BRANCH=enh/noid/add-beta-channel-advice \
-p 8443:443 \
-e TRUSTED_DOMAIN=192.168.146.128 \
--name nextcloud-easy-test \
ghcr.io/szaimen/nextcloud-easy-test:latest
```

</details>

### CODE REVIEW

**Review by Pytal — APPROVED** [review]

File: apps/updatenotification/src/components/UpdateNotification.vue [hunk 1]
```diff
@@ -84,6 +84,11 @@
 			</template>
 		</div>
 
+		<div>
+			{{ t('updatenotification', 'You can change the update channel below which also affects the apps management page.') }} <br>
+			{{ t('updatenotification', 'E.g. after switching to the beta channel, beta app updates will be offered to you in the apps management page.') }}
```
Comment (nickvergessen) [inline apps/updatenotification/src/components/UpdateNotification.vue:89]:
```suggestion
			{{ t('updatenotification', 'You can change the update channel below which also affects the apps management page. E.g. after switching to the beta channel, beta app updates will be offered to you in the apps management page.') }}
```
Should be once sentence so it's linked for translators

**Review by artonge — APPROVED** [review]

File: apps/updatenotification/src/components/UpdateNotification.vue (hunk 1 above)
Comment (nickvergessen) [reply apps/updatenotification/src/components/UpdateNotification.vue:89]: > Good point but I will insert the br tag then...

Please don't add html to the translation. It is not needed to force a line break there.

Comment (szaimen) [reply apps/updatenotification/src/components/UpdateNotification.vue]: addressed

**Pull request merged**